docs/packaging-applications/build-servers/tfs-azure-devops/using-octopus-extension/build-versions-in-team-build.md

```diff
@@ -1,15 +1,15 @@
 ---
-title: Build versions and packaging in Team Build
-description: A guide to build version numbers in Azure DevOps and Team Foundation Server (TFS), and how they work with packages for Octopus.
+title: Build versions and packaging in Azure DevOps
+description: A guide to build version numbers in Azure DevOps, and how they work with packages for Octopus.
 ---
 
-Correctly versioning the packages you deploy with Octopus Deploy is important so the right version gets deployed at the right time. With Azure DevOps and Team Foundation Server (TFS), specifying a package version isn't always straightforward.
+Correctly versioning the packages you deploy with Octopus Deploy is important so the right version gets deployed at the right time. With Azure DevOps, specifying a package version isn't always straightforward.
 
-This guide shows you how best to version your builds and packages in Azure DevOps and TFS, whether you're using [OctoPack](#versioning-when-using-octopack), or the [Octopus Extension in the Visual Studio Marketplace](#versioning-when-using-the-octopus-build-tasks).
+This guide shows you how best to version your builds and packages in Azure DevOps, when using the recommended [Archive Files](http://go.microsoft.com/fwlink/?LinkId=809083) task.
 
 ## Build numbers in Team Build
```

Comment:
Team Build changed to Azure DevOps, should this heading also change?

Reply:
Fixed